pkg/server/event_handler_test.go

```diff
@@ -97,45 +98,87 @@ func TestEventHandler_HttpHandler(t *testing.T) {
 	}{
 		{
 			name: "success",
-			pubsubMessageBytes: []byte(`
+			pubsubMessageBytes: []byte(fmt.Sprintf(`
```

Comment:
I think it would be clearer if this was a dedicated struct with a helper to serialize it, like:

```go
pubsubMessageBytes: toJSON(t, &MyStructure{
  Message: &Message{
    Attributes: &Attributes{},
  },
})

// ...

func testToJSON(tb testing.TB, in any)[]byte {
  tb.Helper()

  b, err := json.Marshal(in)
  if err != nil {
    tb.Fatal(err)
  }

  return b
}
```

If using a struct is too heavy, you can use a `map[any]any`:

```go
pubsubMessageBytes: toJSON(t, map[string]any{
  "message": map[string]any{
    "attributes": map[string]any{
      "bucketId": "foo",
    },
    "data": []byte(`{...}`)
  },
})
```

You don't have to b64 encode the data if it's a structure and `[]byte`.

Reply:
Updated.